src/main/resources/com/google/api/codegen/java/sample.snip

```diff
@@ -39,6 +39,9 @@
   public void run() throws IOException, GeneralSecurityException {
     @if sampleBody.hasInputRequest
       // TODO: Assign values to the desired fields of `{@sampleBody.requestBodyVarName}`.
+      @if sampleBody.methodVerb == "PATCH"
```

Comment:
Change comments to headline style for consistency with others ("Assign values to desired fields", "Only assigned fields will be changed").